backend/pick-git/src/test/java/com/woowacourse/pickgit/acceptance/tag/TagAcceptanceTest.java

```diff
@@ -104,10 +135,13 @@ void extractLanguageTags_InvalidAccessToken_ExceptionThrown() {
         String oauthCode = "1234";
         String accessToken = "oauth.access.token";
 
-        OAuthProfileResponse oAuthProfileResponse = new OAuthProfileResponse(
-            "jipark3", "image", "hi~", "github.com/",
-            null, null, null, null
-        );
+        OAuthProfileResponse oAuthProfileResponse = OAuthProfileResponse
+            .builder()
+            .name("jipark3")
+            .image("image")
+            .description("hi~")
+            .githubUrl("htts://www.github.com/")
+            .build();
 
         // mock
         when(oAuthClient.getAccessToken(oauthCode)).thenReturn(accessToken);
```

Comment:
```suggestion
        given(oAuthClient.getAccessToken(oauthCode)).willReturn(accessToken);
```

given()을 사용하도록 변경해주세요!

Reply:
아이고..ㅋㅋㅋ 주의 하겠습니다!